modules/ROOT/pages/dataweave-formats-json.adoc

```diff
@@ -292,7 +292,7 @@ exception.
 better readability or to compress the JSON into a single line.
 Valid values are `true` or `false`.
 | `skipNullOn` | `String` | `null` | Skips `null` values in the specified data
-structure. By default, it does not skip the values. Valid values are `elements`, `attributes`,
+structure. By default, it does not skip the values. Valid values are `arrays`, `objects`,
 or `everywhere`.
 
 * `arrays` +
```

Comment:
In this next part when each valid values ("arrays", "objects", and "everywhere") are described. It would be nice to change to:

arrays
Ignore and omit null values inside arrays from the JSON output, for example, output application/json skipNullOn="arrays".

objects
Ignore key:value pairs that have a null as the value, for example, output application/json skipNullOn="objects".

Reply:
done